.github/workflows/lint-build-test.yaml

```diff
@@ -0,0 +1,16 @@
+name: Lint, build and test
```

Comment:
I see the lint and build commands, but no testing command

Reply:
Yea I played around with running the e2e tests as part of the pipeline but got stuck. I wanted to finish up the other PR first before continuing on that.